packages/@aws-cdk/aws-chatbot/README.md

```diff
@@ -34,7 +34,13 @@ const slackChannel = new chatbot.SlackChannelConfiguration(this, 'MySlackChannel
   slackChannelId: 'YOUR_SLACK_CHANNEL_ID',
 });
 
-slackChannel.addLambdaInvokeCommandPermissions();
+const invokeCommandStatement = new iam.PolicyStatement({
+  effect: iam.Effect.ALLOW,
+  resources: ["RESOURCE_ARNS_ALLOWED"],
```

Comment:
NitPick: use single quotes to match the general repo style. (I prefer double but what can ya do 🤷🏻‍♂️ )

Reply:
My bad. Will change this :)